dev-ruby/mime/metadata.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="project">
+    <email>ruby@gentoo.org</email>
+    <name>Gentoo Ruby Project</name>
```

Comment:
We can't add packages with others listed as maintainers. Seems like ruby hasn't responded to this PR so they're probably not aware of this addition. You'd need their consent on being added here.

For now, you'll have to become the maintainer if you want it added, but if you don't feel like maintaining it properly, just drop it here. We don't want to force people to become maintainers, but there's no other way around it here.
https://projects.gentoo.org/qa/policy-guide/maintainer.html#pg0602
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide#Proxied_maintainer_in_metadata.xml